tox.ini

```diff
@@ -10,7 +10,7 @@ deps =
   -r test-requirements.txt
 
 [testenv:py3]
-commands = nosetests -v -s --nologcapture tests
```

Comment:
This convenient why remove it ?

Reply:
it was needed to debug the tests when they were failing.